devito/propagator.py

```diff
@@ -55,6 +55,7 @@ class Propagator(object):
                            number used for all dimensions except inner most or a list
                            explicitly stating block sizes for each dimension
                            Set cache_blocking to None to skip blocking on that dim
+                           Set cache_blocking to 0 to use best guess based on architecture
```

Comment:
We've dropped this capability (because it was broken), so you can also drop this line